instructor-info/index.html

```diff
@@ -0,0 +1,10 @@
+---
+layout: page
+title: Instructor Info
+---
+
+<h2>Instructor Infomation</h2>
+
+<ul>
+  <li class="instructor-info"><a href="instructor-info.html">Instructor Info</a></li>
```

Comment:
I think instead of linking to markdown, it would be better to just put the instructor-info content in this index. No need to make people travel another layer, since everything they need to know can go in that first page